crates/bindings/src/table.rs

```diff
@@ -98,14 +89,8 @@ pub trait Table: TableInternal {
     /// Note that as of writing deletion is infallible, but future work may define new constraints,
     /// e.g. foreign keys, which cause deletion to fail in some cases.
     /// If and when these new constraints are added,
-    /// we should define `Self::try_delete` and `Self::TryDeleteError`,
-    /// analogous to [`Self::try_insert`] and [`Self::TryInsertError`].
-    ///
-    /// Note on MVCC: the return value means that logically a `delete` performs a query
-    /// to see if the row is present.
-    /// As such, two concurrent transactions which delete the same row
-    /// cannot be placed in a sequentially-consistent ordering,
-    /// and one of them must be retried.
+    /// we should define `Self::ForeignKeyViolation`,
+    /// analogous to [`Self::UniqueConstraintViolation`].
```

Comment:
Please remove this section; I wrote it as internal docs and did not intend for it to become public.